yaml/ansible.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: APEL for Indigo
+  hosts: localhost
+  tasks:
+    # tasks file for ansible-dockerized-apel-db
```

Comment:
What's a tasks file?

Reply:
to be honest, I'm not sure. I just followed the example [here](https://www.ansible.com/blog/six-ways-ansible-makes-docker-compose-better)

Reply:
Just call them "tasks" then as that seems to be the correct Ansible terminology.

Reply:
changed